cli/cmd/createMi.go

```diff
@@ -1,156 +0,0 @@
-// Copyright (c) Microsoft Corporation.
-// Licensed under the MIT License.
-
-package cmd
```

Comment:
Added the following:
> This also includes updates to the Managed Identity documentation to help improve clarity of the feature. As part of this I deleted the tool `createMi.go` as all it really did was run 3 az cli commands to create a managed identity and assign it permissions. It made assumptions about what permissions you wanted the identity to have. Additionally it suggested to install `aad-pod-identity`, but this guidance is not appropriate for installations via Helm because Helm already installs that dependency as part of the chart. Rather than having a script try to be one-size fits all it makes more sense to give customers the instructions they need to create the managed identity and allow them to choose what permissions they want to give it.

Reply:
Awesome, thanks!